### PR TITLE
Make Event Duration and Time optional

### PR DESCRIPTION
**Pull request by tyanhan**
Also moved the relevant Commands to Person package and Parsers to Person package.

### CODE REVIEW

File: src/main/java/seedu/address/model/schedule/Event.java [hunk 1]
```diff
@@ -84,7 +99,8 @@ public boolean equals(Object other) {
 
     @Override
     public String toString() {
-        return String.format("%s %s %s %s", eventDescription, date, time, duration);
+        return String.format("%s %s %s-%s", eventDescription,
```
Comment (vishandi) [inline src/main/java/seedu/address/model/schedule/Event.java:102]: Is it better if the full day event or events that ends the next day handled separately?

Comment (tyanhan) [reply src/main/java/seedu/address/model/schedule/Event.java:102]: Yup you are right, I was thinking of how to best display this as well.

Comment (vishandi) [reply src/main/java/seedu/address/model/schedule/Event.java:102]: Perhaps if it is on the next day, you could add (+1) at the end to indicate it is the next day? Cos we are avoiding displaying the date agiain in the end right

**Comment by hanqinilnix** [comment]
Next time put a commit for moving classes to other/new packages. Didn't know you moved them since the title and commits didn't suggest it

File: src/main/java/seedu/address/model/schedule/Event.java (hunk 1 above)
Comment (tyanhan) [reply src/main/java/seedu/address/model/schedule/Event.java:102]: Okay sounds good, I will do that.

**Comment by vishandi** [comment]
LGTM!

**Pull request merged**